experimental/dds/tree2/src/feature-libraries/mitigatedChangeFamily.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	ChangeFamily,
+	ChangeFamilyEditor,
+	ChangeRebaser,
+	Delta,
+	TaggedChange,
+	emptyDelta,
+} from "../core";
+
+/**
+ * Makes a given `ChangeFamily` safer to use by wrapping some of its functions in try-catch blocks.
+ *
+ * Mitigated functions:
+ * - {@link ChangeFamily.intoDelta} (an empty Delta is returned instead)
+ * - {@link ChangeRebaser.rebase} (the given `fallbackChange` is returned instead)
+ * - {@link ChangeRebaser.invert} (the given `fallbackChange` is returned instead)
+ * - {@link ChangeRebaser.compose} (the given `fallbackChange` is returned instead)
+ *
+ * @param unmitigatedChangeFamily - The change family to mitigate.
+ * @param fallbackChange - A changeset to use as a fallback when one of the mitigated functions throws.
+ * @param onError - A callback invoked for each error thrown.
+ * @returns a mitigated change family.
+ */
+export function makeMitigatedChangeFamily<TEditor extends ChangeFamilyEditor, TChange>(
+	unmitigatedChangeFamily: ChangeFamily<TEditor, TChange>,
+	fallbackChange: TChange,
+	onError: (error: unknown) => void,
+): ChangeFamily<TEditor, TChange> {
+	return {
+		buildEditor: (changeReceiver: (change: TChange) => void): TEditor => {
+			return unmitigatedChangeFamily.buildEditor(changeReceiver);
+		},
+		intoDelta: (change: TaggedChange<TChange>): Delta.Root => {
+			try {
+				return unmitigatedChangeFamily.intoDelta(change);
+			} catch (error: unknown) {
```

Comment:
Yes, the class of errors is "we cannot make sense of these changes in the current context". The contract for those functions makes them exception-safe, so it's fine to resume from this point on.

As discussed in standup, there are cases where this could lead to divergence between peers. I have changed the policy to always throw and documented what that means.